Modules/Microsoft365DSC/DSCResources/MSFT_AADIdentityAPIConnector/MSFT_AADIdentityAPIConnector.schema.mof

```diff
@@ -0,0 +1,28 @@
+[ClassVersion("1.0.0")]
+class MSFT_AADIdentityAPIConnectionCertificate
+{
+    [Write, Description(""), EmbeddedInstance("MSFT_Credential")] String Pkcs12Value;
```

Comment:
Make sure to provide a description for all properties.

Reply:
Added Description